src/c++/devices/flushnop/dmFlush.c

```diff
@@ -294,8 +294,8 @@ static struct target_type flushTargetType = {
 /**********************************************************************/
 int __init flushInit(void)
 {
-  STATIC_ASSERT(offsetof(FlushDevice, dev)
-                == offsetof(CommonDevice, dev));
+  BUILD_BUG_ON(offsetof(FlushDevice, dev)
+               != offsetof(CommonDevice, dev));
```

Comment:
If you're going to unwrap the line, I  think you should unwrap the line in all five test devices.